Sources/Queues/QueueWorker.swift

```diff
@@ -122,17 +127,15 @@ public struct QueueWorker: Sendable {
         queue: any Queue,
         error: (any Error)? = nil
     ) {
-        // Checks how long the job took to complete
         Timer(
-            label: "\(jobName).jobDurationTimer",
+            label: "\(jobName).duration.timer",
```

Comment:
I think this would break integrations. SemVer wise it's technically fine but it could cause issues. We're not actually changing any information here right? Though the display unit was incorrect before right?

@maciejtrybilo you're using this correct?

Reply:
Changed the name back. The display unit is probably worth keeping as seconds is just not a great metric for job duration IMO